integration-api.md

```diff
@@ -2,3 +2,28 @@
 permalink: /integration-api.html
 ---
 
+# Loc-I Integrated API
+
+[Link to API here](https://api2.loci.cat).
+
+## What
+
+A REST API to a Loc-I Integrated Cache.
+     
+## Why
+
+Provides developers and technical analysts with interfaces to the Loc-I integrated cache to build applications and query a Loc-I cache without having to know SPARQL. This allows rapid development of applications using the REST API.
+
+## How 
+
+Via the REST API. Swagger doc available [here](https://api2.loci.cat/api/v1/doc).
+
+     
+## MVP 
+
+![Loc-I Technical Architecture Overview](images/loci-integrated-api-1.PNG "Loc-I Technical Architecture Overview")
+
+
+## Anticipated future functionality
+
+* Added API endpoints to handle "L-shaped" and "U-shaped" crosswalks between geographies
```

Comment:
What are these?

Reply:
We've used this term internally in the project and it isn't well documents. We need to document this.

Started to do that in https://github.com/CSIRO-enviro-informatics/loci-testdata/wiki/Test-Case-U

Will need to create a separate issue for documenting this.

Reply:
Some documentation about a crosswalks would need to include a description of "spatial hierarchies" that is geographies which are made from more basic "building block" geographies so for example the ASGS datasets States, SA1-4 can, geographically, be exactly described by collections of "building block" or "base units" namely meshblocks. Different spatial hierarchies exists at the moment we have Geofabric and ASGS spatial hierarchies. A U shaped query finds overlaps / contains and within relationships between high level geometries in the hierarchies by querying intersection relationships stored between "base unit" geometries, i.e linksets.